main.js

```diff
@@ -90,6 +89,48 @@ const renderMainWindow = () => {
         });
     }
   })();
+  
+  //read file and show it in app
+  ipcMain.handle('request-load-text', () => {
+    return fs.readFileSync(filePath, "utf8");
+  });
+
+  function getMasterPassword() {
+    return JSON.parse(fs.readFileSync(passwordFilePath, 'utf-8'));
+  }
+  
+  ipcMain.on('save-and-encrypt-text', async (event, text) => {
+    const userFilesDir = path.dirname(process.env.USER_FILE_PATH);
+    const filePath = path.resolve(process.env.USER_FILE_PATH);
+    const encryptedFilePath = path.join(userFilesDir, 'encrypted.txt');
+  
+    try {
+      fs.writeFileSync(filePath, text, 'utf8');
+      console.info(`File saved at: ${filePath}`);
+  
+      const passwordData = getMasterPassword();
+      const { hashedPassword } = passwordData;
+  
+      const encryptedText = sjcl.encrypt(hashedPassword, text);
+  
+      fs.writeFileSync(encryptedFilePath, encryptedText, 'utf8');
+      console.info(`Encrypted file saved at: ${encryptedFilePath}`);
+  
+      dialog.showMessageBox({
+        type: 'info',
+        title: 'Save and Encrypt Successful',
+        message: 'The text has been saved and encrypted successfully!'
+      });
+    } catch (error) {
+      console.error('Error during save and encryption:', error);
```

Comment:
it's better to add date time which i mentioned before, and of course it has to be special function to generate date